### PR TITLE
Display task status in ugliest possible way

### PR DESCRIPTION
**Pull request by dmohs**
Depends on: https://github.com/broadinstitute/firecloud-orchestration/pull/100

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs [hunk 1]
```diff
@@ -234,6 +234,28 @@
...
+    :inputs {"CancerExomePipeline_v2.M2.intervals" "gs://cancer-exome-pipeline-demo-data/panel_100_genes.interval_list"
+             "CancerExomePipeline_v2.M2.tumor_bam" "gs://cancer-exome-pipeline-demo-data/HCC1143_BL.100_gene_250bp_pad.bam"
+             "CancerExomePipeline_v2.M2.normal_bam" "gs://cancer-exome-pipeline-demo-data/HCC1143.100_gene_250bp_pad.bam"
+             "CancerExomePipeline_v2.M2.ref_fasta_dict" "gs://cancer-exome-pipeline-demo-data/Homo_sapiens_assembly19.dict"
+             "CancerExomePipeline_v2.M2.tumor_bai" "gs://cancer-exome-pipeline-demo-data/HCC1143_BL.100_gene_250bp_pad.bai"
+             "CancerExomePipeline_v2.M2.normal_bai" "gs://cancer-exome-pipeline-demo-data/HCC1143.100_gene_250bp_pad.bai"
+             "CancerExomePipeline_v2.M2.ref_fasta" "gs://cancer-exome-pipeline-demo-data/Homo_sapiens_assembly19.fasta"
+             "CancerExomePipeline_v2.M2.ref_fasta_fai" "gs://cancer-exome-pipeline-demo-data/Homo_sapiens_assembly19.fasta.fai"
+             "CancerExomePipeline_v2.M2.m2_output_vcf_name" "mutations.vcf"
+  },
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs]: Bad copy/paste?

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs [hunk 2]
```diff
@@ -87,13 +110,30 @@
...
-                (filter-workflows (:active-filter @state) (:workflows props)))}]])})
+                   (filter-workflows (:active-filter @state) (:workflows props)))}]])
+   :render-workflow-details
+   (fn [{:keys [props state]}]
+     [:div {}
+      [:div {}
+       (style/create-link
+        #(swap! state dissoc :selected-workflow)
+        "Workflows")
+       " > "
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs]: I greatly prefer the `:angle-right` icon, or maybe the HTML right arrow.

Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs]: Good call. Done.

File: src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs (hunk 1 above)
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs]: Non-keywords? I think those just make more sense as raw strings.

Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs]: No, the floating "},"

Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs]: Oh whoops! Yuk! Thank you.

**Comment by MattPutnam** [comment]
:+1: Works for me.  This will be a great launching point for future UI work.

**Pull request merged**